src/semantic-ui/site/elements/container.overrides

```diff
@@ -268,3 +268,16 @@ Container Overrides - REACT-INVENIO-APP-ILS
     }
   }
 }
+
+.ui.container.link-like {
```

Comment:
why did you create this dependance on the container for your styling? are there already existing classes you could use?

Reply:
The idea of the show All is to have the appearance of the Link from react-router-dom. Using link is a bad idea, as even if I use as target the page itself (#) the view will move to the top of the page, instead of the top of the element we just refreshed.

Reply:
I am not sure if it answers my question. Why is this change placed in the `.ui.container`? The style of this component should not be depending on where it is placed (in this case - the container)